Gruntfile.js

```diff
@@ -2,21 +2,16 @@
 
 module.exports = function (grunt) {
     grunt.initConfig({
-        jshint: {
-            src: ["src/**/*.js", "tests/**/*.js"],
-            buildScripts: ["Gruntfile.js"],
-            options: {
-                jshintrc: true
-            }
+        eslint: {
+            src: ["./src/**/*.js", "./tests/**/*.js", "./*.js"]
         },
         jsonlint: {
-            src: ["src/**/*.json", "tests/**/*.json"]
+            src: ["src/**/*.json", "tests/**/*.json", "./*.json"]
         }
     });
 
-    grunt.loadNpmTasks("grunt-contrib-jshint");
+    grunt.loadNpmTasks("grunt-eslint");
```

Comment:
When you have a moment, please move globally to fluid-grunt-eslint so that the reviewer can have the confidence that something is actually being linted